datasets/nasa-usgs-europa-observations.yaml

```diff
@@ -0,0 +1,39 @@
+Name: NASA / USGS Europa Controlled Observations
+Description: 
+Documentation: https://stac.astrogeology.usgs.gov/docs/data/jupiter/europa/galileo_individual_images/
+Contact: https://answers.usgs.gov/
+ManagedBy: "[United States Geological Survey](https://www.usgs.gov)"
```

Comment:
The provider will need to be NASA in this case since this is supported through the Space Act collaboration.

Reply:
Updated to the NASA.gov homepage.